app/controllers/products_controller.rb

```diff
@@ -3,4 +3,7 @@ def index
         @products = Product.all
         @categories = Category.all
     end
+
+  def show
+  end
```

Comment:
we have removed this show method all together and tests are going green, this was an unneeded method.

Reply:
Actually, I think that you need this method since Cartify depends on having a path to show products. I'm surprised that the tests are going green though...

Reply:
The next User story is about User reviewing his products, so that will be when we need to add the show method.